techhive-ui/src/components/Login/Login.css

```diff
@@ -0,0 +1,10 @@
+.alert {
+    color: red;
+    margin-top: 10px;
+    /* Add more styles as needed */
```

Comment:
we don't need to commit these instructional comments

Reply:
Okay noted. I thought I took them out. I will go through all my code and take it out with the console logs